taxi/forms.py

```diff
@@ -0,0 +1,68 @@
+import re
+
+from django import forms
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    license_number = forms.CharField(
+        max_length=8,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message=(
+                    "The license number must consist of 8 characters: "
+                    "the first 3 characters are capital letters, "
+                    "the last 5 characters are numbers."
+                )
+            )
+        ]
+    )
+
+    class Meta:
+        model = Driver
+        fields = ["license_number"]
+
+
+class DriverCreateForm(forms.ModelForm):
+    password = forms.CharField(widget=forms.PasswordInput)
+
+    class Meta:
+        model = Driver
+        fields = [
+            "username",
+            "first_name",
+            "last_name",
+            "license_number",
+            "password"
+        ]
+
+    def clean_license_number(self):

```

Comment:
why not inherit DriverLicenseForm - then you will not have to define license validation twice